src/renderer/components/icon/icon.tsx

```diff
@@ -78,8 +128,8 @@ export interface BaseIconProps {
 export interface IconProps extends React.HTMLAttributes<any>, BaseIconProps {}
 
 export function isSvg(content: string): boolean {
-  // data-url for raw svg-icon
-  return String(content).includes("svg+xml");
+  // source code of the asset
+  return String(content).includes("<svg");
```

Comment:
This code was introduced in #4725, but was accidentally a breaking change since users of `<Icon>` were previously permitted to pass in SVG XML and not a data URL.